test/patterns/multiple_while/test_multiple_while.py

```diff
@@ -33,7 +33,37 @@ def test_simple(self):
             Path(
                 Path(
                     os.path.realpath(__file__)
-                ).parent, 'MultipleWhile.java'
+                ).parent, 'Simple.java'
+            )
+        )
+        self.assertEqual(lines, [2])
+
+    def test_one_while(self):
+        lines = MultipleWhile().value(
+            Path(
```

Comment:
Here and everywhere else we better write
```python
Path(__file__).parent.absolute() / 'OneWhile.java'
```

Reply:
ok